rules/pre-mehari/snakefiles/scripts/generate_manifest.py

```
@@ -0,0 +1,145 @@
+import sys
+from pathlib import Path
+
+
+def count_lines(tsv_file):
+    """Count lines in a TSV file, excluding the header."""
+    try:
+        with open(tsv_file, 'r') as f:
+            # Skip header and count remaining lines
+            lines = sum(1 for _ in f) - 1
+            return max(0, lines)  # Ensure non-negative
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)